merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -154,15 +168,16 @@ public List<String> validateModelArguments(final String missionModelId, final Ma
          MissionModelLoadException,
          InvalidArgumentsException
   {
-    return this.loadConfiguredMissionModel(missionModelId)
-               .validateConfiguration(arguments);
+    return this.loadUnconfiguredMissionModel(missionModelId)
+               .validateMissionModelArguments(arguments);
```

Comment:
This is the only use of the `validateMissionModelArguments` method. Given that it's a one-liner, should we inline it here and remove it from `MissionModelFacade`?